app/utils/ProcessUtils.scala

```diff
@@ -16,53 +16,28 @@
 
 package utils
 
-import java.util.UUID
+
 
 import models._
 import models.errors.{BadRequestError, Errors}
 import models.ocelot.Process
-import models.ocelot.stanzas.PageStanza
 import play.api.Logger
 import play.api.libs.json.{JsError, JsObject, JsSuccess}
 
 object ProcessUtils {
 
   val logger = Logger(getClass)
 
-  def validateProcess(jsonProcess: JsObject): RequestOutcome[Process] = {
+  def validateProcess(jsonProcess: JsObject): RequestOutcome[Process] =
     jsonProcess.validate[Process] match {
       case JsSuccess(process, _) =>
         Right(process)
       case JsError(errors) =>
         logger.error(s"Parsing process failed with the following error(s): $errors")
         Left(Errors(BadRequestError))
     }
-  }
 
-  def createApprovalProcess(id: String, title: String, status: String, jsonProcess: JsObject): ApprovalProcess = {
+  def createApprovalProcess(id: String, title: String, status: String, jsonProcess: JsObject): ApprovalProcess =
```

Comment:
As there is currently only one usage of this method does it really need to be here?

Reply:
agree